### PR TITLE
Move `master` to `main`

### PR DESCRIPTION
**Pull request by jakirkham**
Partially addresses ( https://github.com/zarr-developers/community/issues/44 )

Switches `.github` links to use `main` (the branch was already changed there).

Also uses relative links internally to avoid referencing the branch at all.

### CODE REVIEW

**Comment by jakirkham** [comment]
One can explore the changes [here]( https://github.com/zarr-developers/governance/tree/6facb62362e184c2f34cc2881ba0c8149516ed40 ).

As most of these were to the `.github` repo that already made the move, those changes were uninteresting.

The more interesting ones are in [ZEP 0]( https://github.com/zarr-developers/governance/blob/6facb62362e184c2f34cc2881ba0c8149516ed40/ZEP/instructions/zep0000.md#discussing-a-zep ) where relative links were used.

Went through and manually checked the links.

File: ZEP/instructions/zep0000.md [hunk 1]
```diff
@@ -395,7 +395,7 @@ This document has been placed in the public domain.
 
 <!-- governance specific links -->
 [CODE OF CONDUCT]: https://github.com/zarr-developers/.github/blob/main/CODE_OF_CONDUCT.md
-[Zarr Steering Council]: https://github.com/zarr-developers/governance/blob/master/GOVERNANCE.md#steering-council
+[Zarr Steering Council]: ../../GOVERNANCE.md#steering-council
```
Comment (joshmoore) [inline ZEP/instructions/zep0000.md:398]:
```suggestion
[Zarr Steering Council]: ../../GOVERNANCE.md#zarr-steering-council
```

**Review by joshmoore — APPROVED** [review]
One minor change to commit but otherwise, :+1:

File: ZEP/instructions/zep0000.md (hunk 1 above)
Comment (jakirkham) [reply ZEP/instructions/zep0000.md]: Oops thank you! 🙏

**Pull request merged**